Bottom: e991bba32df9df582f8f48dea740a07b91e5e9d9
Top:    7dc5e5bcab773a09880cf99fc2a43a97da9827d4
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 16:15:38 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index 23cc284..ae1caf7 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -29,8 +29,8 @@
 #include <sys/vfs.h>
 #include <sys/utsname.h>
 #include <netinet/in.h>
-#include <linux/fs.h>
 #include <sys/mount.h>
+#include <linux/fs.h>
 
 #define MODULE_PARSE
 #include "automount.h"
